src/ui/layout.rs

```diff
@@ -247,6 +247,22 @@ impl Layout {
             None
         }
     }
+
+    /// Propagate the given event to the command line.
+    fn command_line_handle_event(&mut self, event: Event) -> EventResult {
+        let is_left_right_event = matches!(event, Event::Key(Key::Left) | Event::Key(Key::Right));
+        let result = self.cmdline.on_event(event);
```

Comment:
Nit: It probably works fine because `result` contains a callback, but I wonder if the call to `self.cmdline.on_event()` should be moved inside the `else` block? Otherwise we pass events to the `cmdline` even if it's a left/right keypress.

Reply:
I think I understand what you mean, but I don't know whether it would be correct. If `self.cmdline.on_event(event)` was moved to the else, the user couldn't move left and right anymore inside the command line.

Reply:
Ah yes, you're right.